config/config.exs

```diff
@@ -57,7 +57,7 @@ config :data_aggregator, DataAggregatorWeb.Gettext, default_locale: "en"
 config :data_aggregator, Oban,
   repo: DataAggregator.Repo,
   plugins: [
-    {Oban.Plugins.Pruner, max_age: 5, limit: 10_000, interval: 1_000 * 60},
+    {Oban.Plugins.Pruner, max_age: 24 * 60 * 60, limit: 10_000, interval: 1_000 * 60},
```

Comment:
keep finished oban jobs for 24h to track down errors from the dashboard instead of crawling through the logs like a mole